doc/neps/nep-0038-SIMD-optimizations.rst

```diff
@@ -180,20 +195,50 @@ Anyone writing a SIMD loop will use the ``npyv_load_u32`` macro instead of the
 architecture specific intrinsic. The code also supplies guard macros for
 compilation and runtime, so that the proper loops can be chosen.
 
+At compile time, the CPU is probed to determine which features are available.
+These are mapped into the CPU-architecture-specific dictionary
+``__cpu_features__``, the values are filled in at run-time by again probing
+the CPU.
+
+Two new build options are available to ``runtests.py`` and ``setup.py build``.
```

Comment:
the new arguments can be reached also from `build_clib` and  `build_ext`,
if `build_clib` or `build_ext` are not specified by the user, the arguments of `build` will be used, which also hold the default values.

Reply:
Adding `build_clib` and `build_ext`. The defaults are explained below.

Reply:
`build_clib` isn't commonly used, but `build_ext` is probably used more often than `build` (for in-place builds). so indeed good to add.